contracts/HenkakuBadge.sol

```diff
@@ -123,6 +143,7 @@ contract HenkakuBadge is ERC1155, Ownable {
             msg.sender == owner() || msg.sender == _of,
             "NOT HAVE AUTHORITY"
         );
+        totalSupply[_tokenId] -= 1;
```

Comment:
I think it is common that burn does not increase the supply. It is burn to increase scarcity to increase price, etc. But this is intentional, right?

Reply:
uh, kinda true. hmm, once we burn we can make new supply. it's matter of spec